configure.ac

```diff
@@ -3454,6 +3454,13 @@ AC_CHECK_FUNCS(alarm accept4 setitimer getitimer bind_textdomain_codeset chown \
  truncate uname unlinkat unsetenv utimensat utimes waitid waitpid wait3 wait4 \
  wcscoll wcsftime wcsxfrm wmemcmp writev _getpty)
 
+# Force lchmod off for Linux. Linux disallows changing the mode of symbolic
+# links. Some libc implementations have a stub lchmod implementation that always
+# returns an error.
+if test "$MACHDEP" = linux; then
+  undef HAVE_LCHMOD
```

Comment:
`./configure: line XXXXXX: undef: command not found`
And do not use shell's `unset` since `HAVE_LCHMOD` here would not be a shell variable anyway.

The solution is probably `AC_DEFINE(HAVE_LCHMOD, 0)` and ensuring that C/C++ code uses something like `#if defined(HAVE_LCHMOD) && HAVE_LCHMOD` instead of just `#ifdef HAVE_LCHMOD` or `#if defined(HAVE_LCHMOD)`.